docs(weave): add .delete() to object doc page (#3452)
wandb/weave: docs/docs/guides/tracking/ops.md

```bash
export WEAVE_PRINT_CALL_LINK=false
```

## Deleting an op

<Tabs groupId="programming-language" queryString>
  <TabItem value="python" label="Python" default>

>     You can call `.delete()` on any op Ref to delete that version of the op.

Reviewer: ```suggestion
     To delete a version of an op, call `.delete()` on the op reference object.
```
